atom/scalars.py

```diff
@@ -176,21 +178,35 @@ def __init__(self, default=0.0, factory=None, strict=False):
             self.set_validate_mode(Validate.FloatPromote, None)
 
 
-class Str(Value):
-    """ A value of type `str`.
-
-    By default, unicode strings will be promoted to plain strings. Pass
-    strict=True to the constructor to enable strict string checking.
+class Bytes(Value):
+    """ A value of type `bytes`.
```

Comment:
You should really comment on the promotion rules here since they also affect this type.